docs/software/cloud.md

```diff
@@ -528,7 +528,7 @@ Credits: This software title has been added to DietPi-Software by CedArctic, man
 
 Bitwarden_RS is an unofficial Bitwarden password manager server with web interface, written in Rust.
 
-![Bitwarden_RS web vault screenshot](../assets/images/dietpi-software-cloud-bitwarden_rs.png)
+![Bitwarden_RS web valut screenshot](../assets/images/dietpi-software-bitwarden_rs.jpg){: style="width:600px"}
```

Comment:
But it's still "vault" 😉.

Please remove the old image then, so we don't have an unused file.

Reply:
Ok. Thx.